pyproject.toml

```diff
@@ -10,61 +10,94 @@ requires = [
 [tool.robotpy-build.static_libs.ctre_api.maven_lib_download]
     artifact_id = "api-cpp"
     group_id = "com.ctre.phoenix"
-    repo_url = "http://devsite.ctr-electronics.com/maven/release"
-    version = "5.18.2"
+    repo_url = "https://devsite.ctr-electronics.com/maven/release"
+    version = "5.19.4"
     libs = ["CTRE_Phoenix"]
 
 [tool.robotpy-build.static_libs.ctre_canutils.maven_lib_download]
     artifact_id = "canutils"
     group_id = "com.ctre.phoenix"
-    repo_url = "http://devsite.ctr-electronics.com/maven/release"
-    version = "5.18.2"
+    repo_url = "https://devsite.ctr-electronics.com/maven/release"
+    version = "5.19.4"
     libs = ["CTRE_PhoenixCanutils"]
 
 [tool.robotpy-build.static_libs.ctre_canutils.override.arch_athena]
     ignore = true
 
 [tool.robotpy-build.static_libs.ctre_cci.maven_lib_download]
+    artifact_id = "cci-sim"
+    group_id = "com.ctre.phoenix.sim"
+    repo_url = "https://devsite.ctr-electronics.com/maven/release"
+    version = "5.19.4"
+    libs = ["CTRE_PhoenixCCISim"]
+
+[tool.robotpy-build.static_libs.ctre_cci.maven_lib_download.override.arch_athena]
```

Comment:
I think we need to switch these around. Simulation is only supported on the desktop OSes. Raspbian and AArch64 Bionic (which I believe is what the Jetson runs?) are expected to use the SocketCAN implementation, from what I can tell.